### ISSUE TITLE
Add possibility to configure which sections will be displayed on line graphs

### ISSUE THREAD

**Issue by WebCloud** [GoogleChrome/lighthouse-ci#158]
RIght now, the project page has four default sections that will be shown as line graphs to follow progression over time. It would be nice to allow the lighthouserc to expose and configure which sections from the report you'd like to keep track of as key metrics for your project.

Specially with the hability to create new sections with stack packs and plugins to monitor pain points specific to the application in quesion, this would be a nice feature to have 😄

**Comment by Zizzamia** [comment]
Great point, @WebCloud what are the specific metrics you felt were more important to keep track of separately?

**Comment by WebCloud** [comment]
Hey @Zizzamia!

One of them is Largest Contentful Paint. But also it would be interesting to see a graph for custom metrics we generate. Cause normally those are very important for the product SLAs on performance and it would be nice to have a visual representation of the progress on those over time.

**Comment by WebCloud** [comment]
At Spotify we are building a reporting script based on the apis from lhci, to report back to the PR comparing the LH data from the PR against master. And one of the things we are trying to keep track on benchmarking against master are those custom perfomrance metrics that we are generating.

**Comment by Zizzamia** [comment]
Yeah, that's dope! I still struggle with creating custom performance metric for Lab, it would be dope have some documentation around that. For now, I just add custom metrics at RUM time with Perfume.
